src/app.ts

```diff
@@ -1,8 +1,56 @@
-import express from 'express'
+const express = require('express')
```

Comment:
I wonder if we shouldn't use import instead of require with typeScript ? (VSCode show me errors if I use require)

Reply:
Good point, let's use `import`